flarefly/data_handler.py

```diff
@@ -15,7 +15,7 @@ class DataHandler:
     Class for storing and managing the data of (ROOT tree, TH1, numpy array, etc.)
     """
 
-    def __init__(self, data=None, var_name='', limits=None, use_zfit=True, **kwargs):
+    def __init__(self, data=None, var_name='', limits=None, use_zfit=True, nbins=100, **kwargs):
```

Comment:
Maybe let's put it in the kwargs
```suggestion
    def __init__(self, data=None, var_name='', limits=None, use_zfit=True, **kwargs):
```